dotnet/src/SemanticKernel.Abstractions/Orchestration/SKContext.cs

```diff
@@ -151,60 +132,90 @@ private string DebuggerDisplay
     /// <summary>
     /// Whether an error occurred while executing functions in the pipeline.
     /// </summary>
-    public bool ErrorOccurred { get; private set; }
-
-    /// <summary>
-    /// Error details.
-    /// </summary>
-    public string LastErrorDescription { get; private set; } = string.Empty;
+    public bool ErrorOccurred => this.LastException != null;
```

Comment:
nit: Maybe we can rename this property to `ExceptionOccurred` for better consistency.